packages/radix-ui-themes/src/components/data-list.css

```diff
@@ -1,9 +1,3 @@
-.rt-DataListItem {
```

Comment:
DId you mean to delete this?

Reply:
Yeah, this stuff is set in the orientation block